module3/lessons/intro-to-cypress-prework.md

```diff
@@ -0,0 +1,437 @@
+---
+title: Intro to Cypress Testing
+length: 1 hour
+tags: Cypress, testing
+module: 3
+---
+<!-- Keep the pre-work separated from the lesson 
+Pre-work needs to be moved to the new curriculum site 
+- I think we make this lesson too long, and pin pointed, part of the pre-work, small react app 
+- ideabox repo few test files and written out and then after they look through the item, this is how we do E2E testing, let us show them 
+look how to typing in the value and look how the post works, and corresponding test suits. 
+before feedback - section some other application and something similar and has the test suit and make sense of it
+now download these 2 repos and setup the cypress and see functional test suits. 
+What si on the page load.
+
+ -->
+## Prework:
```

Comment:
Can we get rid of this "prework" header and add a line stating,  as prework for the Intro to Cypress lesson, work through this entire Prework lesson, submit the deliverable, write down questions you have and bring them to the Intro to Cypress class.

I worry that the Prework header might confuse people into thinking that they dont need to complete the entire lesson